commands.js

```diff
@@ -272,6 +272,26 @@ exports.commands = {
 	},
 	backhelp: ["/back - Unblocks challenges and/or private messages, if either are blocked."],
 
+	rank: function (target, room, user) {
+		if (!target) target = user.name;
+
+		let buffer = '<div class="ladder"><table>';
+		buffer += '<tr><td colspan="8">User: <strong>' + Tools.escapeHTML(target) + '</strong></td></tr>';
+
+		Ladders.Ladder.visualizeAll(target).then(values => {
```

Comment:
You might want to export `visualizeAll` directly from `Ladders` so you don't have to do `Ladders.Ladder`.